### ISSUE TITLE
tif to avif convert, green / pink color distorted

### ISSUE THREAD

**Issue by Simon566** [ImageMagick/ImageMagick#6111]
### ImageMagick version

7.1.0-62 HDR

### Operating system

Windows

### Operating system, version and so on

Win 11 latest as of today

### Description

when converting from TIF to avif with "mogrify -format avif -depth 8 -quality 75 *.tif" i recognize color distortion.

converting the same image to jpg doesnt show this

please check the green grass color between TIF and avif/jpg , or the pink color of the rider shirt

### Steps to Reproduce

convert the TIF to avif / jpg with mogrify , check the color difference

### Images

[dsc_1662.zip](https://github.com/ImageMagick/ImageMagick/files/10840041/dsc_1662.zip)

**Comment by fmw42** [comment]
You are on IM 7. Use magick mogrify not mogrify. The latter is IM 6 syntax and the former is IM 7 syntax.  For a single image, I suggest using magick, not magick mogrify.

I did the following on IM 7.1.0.62 on Mac OSX Monterey

```
 magick dsc_1662.TIF -depth 8 -quality 75 dsc_1662.avif
 magick dsc_1662.TIF -depth 8 -quality 75 dsc_1662.jpg
```

Both images are the same.

I also did

```
magick mogrify -format avif -depth 8 -quality 75 dsc_1662.TIF
magick mogrify -format jpg -depth 8 -quality 75 dsc_1662.TIF
```

And both images are the same.

What viewer did you use?

**Comment by Simon566** [comment]
Hi , 

i will test and report the outcome.

Just a question: when i use IM6 syntax , is it executing different code and then the result is different ? mogrify.exe is something else then magick.exe mogrify whatsoever ?

regards,
Simon

**Comment by fmw42** [comment]
I do not think you will see much difference. But try magick mogrify and see if that fixes it.  Otherwise, check your version of libtiff (magick -list format will tell you the version).

**Comment by fmw42** [comment]
Both your provided images display the same for me.  I suspect your issue is with your viewer.

**Comment by fmw42** [comment]
magick compare -metric rmse dsc_1662.avif dsc_1662.jpg null:
1845.87 (0.0281662)

This is less than 3% different, which considering jpg compression, tells me that they are the same.

**Comment by snibgo** [comment]
Viewing the zipped images from the OP the JPG, AVIF and TIFF look identical to me when using Microsoft Windows 11 "Photos", or IrfanView "XnView MP".

**Comment by Simon566** [comment]
are you using the Microsoft AV1 codec pack for the photo gallery as well ? Irfanview also seems to use it

**Comment by Simon566** [comment]
left to right , jpg , tiff , avif .

do you see the green being darker in the avif ?
![Screenshot 2023-02-28 131936](https://user-images.githubusercontent.com/43811457/221852687-aec15be2-079d-4701-bed9-08aadd92ea4d.png)

**Comment by snibgo** [comment]
> are you using the Microsoft AV1 codec pack for the photo gallery as well ?

I don't know. How can I find out?

> do you see the green being darker in the avif ?

No, I don't see that. The avif version has higher local contrast than the tiff version, but it seems neither lighter or darker. Viewing that PNG image in Gimp and using the dropper tool with a large sample (24x24 pixels), and trying to choose the same areas, shows no significant difference.

**Comment by Simon566** [comment]
im not sure . i deinstalled the av1 codec from microsoft and IrfanView doesnt show the pic anymore + Windows Photo gallery tells me only a lowres preview is to be seen ( with the distorted colors )

gimp looks ok.

so if you ask me, the av1 microsoft codec is to be blamed.

which codecs are you using for windows 10/11 to have the photo app working ?

**Comment by ValZapod** [comment]
>Photo gallery with AV1 codec

Why? Use mpv or ffplay.
